share/spice/rand_word/data/single_word.handlebars

```diff
@@ -0,0 +1,2 @@
+<a href="http://wordnik.com/words/{{word}}">{{word}}</a>
```

Comment:
Do we need the handlebars template anymore? Can we leverage a `title` and `subtitle` setting in the javascript here?

Similar to https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/xkcd/display/xkcd_display.js